Packs/Core/Playbooks/playbook-Suspicious_access_to_shadow_file_README.md

```diff
@@ -0,0 +1,67 @@
+This playbook addresses the following alerts:
+ 
+- Uncommon creation or access operation of sensitive shadow copy by a high-risk process
+- Suspicious access to shadow file
+ 
+Playbook Stages:
+  
+Triage: 
+ 
+- <Action description>
+ 
+Early Containment:
+ 
+- <Action description>
+ 
+Investigation:
+ 
+- <Action description>
+ 
+Containment:
+ 
+- <Action description>
```

Comment:
Missing info

Reply:
Sorry, I assigned it to you when still in draft, now I've updated the PR with the missing info - Thanks!